estela-web/src/components/JobDetailPage/index.tsx

```diff
@@ -253,192 +609,620 @@ export class JobDetailPage extends Component<RouteComponentProps<RouteParams>, J
+                                        >
+                                            {envVar.name}: {envVar.value}
+                                        </Tag>
+                                    ))}
+                                </Space>
+                            </Col>
+                        </Row>
+                        <Row className="grid grid-cols-3 bg-estela-blue-low py-1 px-2 rounded-lg">
+                            <Col>
+                                <Text className="font-bold">Arguments</Text>
+                            </Col>
+                            <Col className="px-2">
+                                <Space direction="horizontal">
+                                    {args.map((arg: ArgsData, id) => (
+                                        <Tag
+                                            className="border-estela-blue-full bg-estela-blue-low text-estela-blue-full rounded-md"
+                                            key={id}
+                                        >
+                                            {arg.name}: {arg.value}
+                                        </Tag>
+                                    ))}
+                                </Space>
+                            </Col>
+                        </Row>
+                        <Row className="grid grid-cols-3 py-1 px-2">
+                            <Col>
+                                <Text className="font-bold">Spider</Text>
+                            </Col>
+                            <Col>
+                                <Link
+                                    to={`/projects/${this.projectId}/spiders/${this.spiderId}`}
+                                    className="text-estela-blue-medium px-2"
+                                >
+                                    {spiderName}
+                                </Link>
+                            </Col>
+                        </Row>
+                    </Card>
+                </Content>
+                <Content className="my-2 grid lg:grid-cols-12 grid-cols-12 gap-1 items-start lg:w-full">
+                    <Card className="w-full col-span-2 flex flex-col" style={{ borderRadius: "8px" }} bordered={false}>
+                        <Text className="py-0 text-estela-black-medium font-medium text-base">Bandwidth</Text>
+                        <Row className="grid grid-cols-1 py-1 mt-3">
+                            <Col>
+                                <Text className="font-bold text-estela-black-full text-lg">
+                                    {requestCountDecimalPercentage.toFixed(2)}
+                                </Text>
+                                <Text className="text-estela-black-full text-base">/1GB</Text>
+                            </Col>
+                            <Col>
+                                <Text className="text-estela-black-medium text-xs">of project (disabled)</Text>
```

Comment:
You can disable the whole `Col` tag, also reduce the opacity to show that feature disabled